sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/Storage/BlobsCheckpointStore.Diagnostics.cs

```diff
@@ -0,0 +1,213 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Messaging.EventHubs.Processor.Diagnostics;
+
+namespace Azure.Messaging.EventHubs.Processor
+{
+    internal sealed partial class BlobsCheckpointStore
```

Comment:
nit:   Please add doc comments.

Reply:
You might want to tweak doc analyzer settings for event hubs to require these. That way compiler would catch it for you :)

Reply:
I like this idea!   How would one go about learning how to do such a thing?